product_multi_company/__manifest__.py

```diff
@@ -7,10 +7,13 @@
     "author": "Tecnativa," "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/multi-company",
     "category": "Product Management",
-    "version": "13.0.1.0.0",
+    "version": "13.0.1.1.0",
```

Comment:
As this is a great change involving migration scripts, you should bump major version

Reply:
I thought ocabot would handle this, but it doesn't rename the migration directory of course.